<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>391</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="7" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Font Size</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEditSize"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelFamily">
      <property name="text">
       <string>Family</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Font Size Type</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="comboBoxEditSize"/>
    </item>
    <item row="2" column="1">
     <widget class="QFontComboBox" name="fontComboBox"/>
    </item>
    <item row="7" column="1">
     <widget class="QSlider" name="sliderEditSize">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Font Size Type</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="AlexLineEdit" name="lineEditFamily"/>
    </item>
    <item row="0" column="1">
     <widget class="AlexLabel" name="labelFox">
      <property name="text">
       <string>The quick brown fox likes CSE 335</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Family</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="radioButtonArial">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Arial</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButtonTNR">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>30</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Times New Roman</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButtonSans">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>30</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Comic Sans MS</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
   <zorder>labelFamily</zorder>
   <zorder>lineEditFamily</zorder>
   <zorder>fontComboBox</zorder>
   <zorder>label_3</zorder>
   <zorder>lineEditSize</zorder>
   <zorder>labelFox</zorder>
   <zorder>sliderEditSize</zorder>
   <zorder>label_2</zorder>
   <zorder>comboBoxEditSize</zorder>
   <zorder>label_4</zorder>
   <zorder>label</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AlexLabel</class>
   <extends>QLabel</extends>
   <header location="global">alexlabel.h</header>
   <slots>
    <slot>setFont(QFont)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>AlexLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">alexlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
